internal/cmdexport/cmdexport.go

```diff
@@ -83,14 +61,37 @@ func GetExportRunner() *ExportRunner {
 
 // The ExportRunner wraps the user's input and runs the command.
 type ExportRunner struct {
-	Orchestrator   string
-	OutputFilePath string
-	Command        *cobra.Command
+	WorkflowOrchestrator string
+	OutputFilePath       string
+	Command              *cobra.Command
 	*types.PipelineConfig
 	Pipeline orchestrators.Pipeline
 }
 
 func (r *ExportRunner) preRunE(cmd *cobra.Command, args []string) (err error) {
+	r.Dir = args[0]
+
+	if len(r.WorkflowOrchestrator) == 0 {
+		return fmt.Errorf("--workflow flag is required")
+	}
+
+	switch r.WorkflowOrchestrator {
+	case "github-actions":
+		{
+			r.Pipeline = new(orchestrators.GitHubActions)
+		}
+	case "cloud-build":
+		{
+			r.Pipeline = new(orchestrators.CloudBuild)
+		}
+	case "gitlab-ci":
+		{
+			r.Pipeline = new(orchestrators.GitLabCI)
+		}
+	default:
+		return fmt.Errorf("unsupported orchestrator %v", r.WorkflowOrchestrator)
```

Comment:
```
unsupported orchestrator %v. It must be one of github-actions, cloud-build and gitlab-ci.
```

Reply:
Maybe put "github-action, cloud-build and gitlab-ci" into a string variable, since it's used several times in the file.

Reply:
Cobra doesn't have a built-in utility for taking in an enum? interesting.